docs/Bare-Metal/ZFS.md

```diff
@@ -344,5 +345,26 @@ proc4 2 187 1724229556
 proc4ops 72 0 0 0 283322 277380 92 2950 0 7998 2976415 159260 147 1471402 0 1115825 178521 0 0 280797 0 0 41 7332238 0 948 1472379 4742 0 123236 789 0 146 936 0 36300 0 0 0 531747 0 0 1 384 213 186 498725 0 0 0 0 0 0 472 7929603 0 0 0 198 193 0 0 0 0 0 0 0 0 0 0 0 0 0
 ```
 
+### Updating ZFS and Centos
+If you need to just update the ZFS version, then a simple command like `yum upgrade zfs`. I would check with someone more knowledgeable first before you do this(Richard/Mike/Dean). In this section, I will discuss how to update a ZFS server that is in production, and what are the best practices to update both the Centos OS and the ZFS version running on it.
+
+#### Pre-Update Tasks
+1. Make sure to schedule a time for the services running on the cluster to be down way in adavance. Inform the users of the downtime, and keep them updated with any updates on the status of the cluster.
+1. It is ***strongly*** to use a pair of new SSDs for the OS instead of just updating the OS on the production pair of SSDs.
+1. Take this opportunity to update any major firmware/driver releases.
```

Comment:
> Take this opportunity to update any major firmware/driver releases.

From someone who hasn't really interacted with the ZFS, is there any important information we should know about firmware/driver releases? Would this include links to the releases or stuff like that, or is it just a simple command?

Reply:
Have added clarification on this with commit b842fb8.